setup.cfg

```diff
@@ -48,3 +48,5 @@ ignore =
     W503
     # N818 Exceptions should be named Error (TODO)
     N818
+    # B028 Suggest !r when formatted value in f-string is surrounded by quotes.
+    B028
```

Comment:
Pre-commit was failing since the latest flake8 update which includes a check for quotes outside of the braces in an f-string (B028). In most cases in our codebase, these quotes are actually necessary, so I suggest to ignore this new warning.